### ISSUE TITLE
Choose a reasonable method to provide support for JSON-RPC.

### ISSUE THREAD

**Issue by feiyu-he** [qingfengzxr/gdscript-web3#13]
Hi, @qingfengzxr, have a nice day!
In Godot game development, developers commonly use the **HTTPRequest** node to handle functionalities related to HTTP requests. **HTTPRequest** operates **asynchronously**, without offering a **synchronous** option. This is reasonable because, in game development, we generally prefer asynchronous processing over synchronous to enhance the gaming experience.
However, achieving synchronous returns is also possible. As a lower-level interface, Godot's HTTPClient offers this capability.
We need to carefully choose whether we require synchronous or asynchronous support for JSON-RPC, and which approach to use (Godot C++ module or direct GDScript usage).

For both methods, I provide sample demos. If you have any questions, you can discuss them under this issue or send me an email.

**Comment by feiyu-he** [comment]
PR aircraft: https://github.com/qingfengzxr/gdscript-web3/pull/14

**Comment by qingfengzxr** [comment]
Indeed, it's needs careful consideration. I will do some research and try it first. Thank you very much~

**Comment by qingfengzxr** [comment]
We chose to support both synchronous and asynchronous methods. However, the asynchronous implementation relies on the functions of the godot framework. I will provide examples and place them blew.
